src/common/services/LearningPath/getLearningPathElement.tsx

```diff
@@ -1,6 +1,15 @@
-import { LearningPath, LearningPathReturn } from '@core'
+import { LearningPathElement, LearningPathElementReturn } from '@core'
 
-export const getLearningPath: LearningPathReturn = async (userId, lmsUserId, studentId, course_id, topic_id) => {
+export const getLearningPathElement: LearningPathElementReturn = async (
+  userId,
+  lmsUserId,
+  studentId,
+  course_id,
+  topic_id
+) => {
+  if (!course_id || !topic_id) {
+    throw new Error('course_id and topic_id are required')
+  }
   return fetch(
```

Comment:
Hab ne neue Syntax für das Returnen von Daten, die die Infrastruktur leichter verständlich macht^^
```suggestion
const response = await fetch(
    process.env.BACKEND +
    `/user/${userId}/${lmsUserId}/student/${studentId}/course/${course_id}/topic/${topic_id}/learningPath`,
    {
      method: 'GET',
      credentials: 'include',
      headers: {
        'Content-Type': 'application/json'
      }
    }
  )
  return getData<LearningPathElement>(response)
```